src/history/__tests__/historyActions.test.ts

```diff
@@ -33,50 +35,67 @@ describe('history actions', () => {
     priceFilter
   })
 
-  beforeEach(() => {
-    store = generateMockStore(mockGlobalState, mockApi)
-    store.clearActions()
-  })
+  const expectedFailedAction = {
+    type: HISTORY_FETCH_FAILED
+  }
 
-  it('should create action update history data', () => {
-    expect(historyReceivedAction(mockHistoryState)).toEqual(expectedHistoryReceivedAction)
-  })
+  describe('fetch success', () => {
+    beforeEach(() => {
```

Comment:
Maybe this should only apply to any tests that uses the store. But the test after this does not have or seem to require the store